client/src/controllers/TooltipController.test.js

```diff
@@ -133,4 +133,26 @@ describe('TooltipController', () => {
 
     expect(tooltip.textContent).toEqual('NEW content!');
   });
+  it('should support passing the offset value', async () => {
+    document.body.innerHTML = `
+    <section>
+      <button
+        id="button"
+        type="button"
+        data-controller="w-tooltip"
+        data-w-tooltip-offset-value="[10, 20]"
+      >
+        CONTENT
+      </button>
+    </section`;
```

Comment:
```suggestion
    </section>`;
```

Watch the closing bracket here, I have a fix coming for the other parts of this test #10885 but we should ensure this new work has the correctly formed HTML.

This won't break the test but it can cause issues with IDE syntax highlighting.